client/my-sites/plugins/index.js

```diff
@@ -35,6 +36,16 @@ module.exports = function() {
 	}
 
 	if ( config.isEnabled( 'manage/plugins' ) ) {
+		page( '/plugins/wpcom-masterbar-redirect/:site', context => {
+			analytics.tracks.recordEvent( 'calypso_wpcom_masterbar_plugins_view_redirect' );
```

Comment:
Oh let's avoid the direct analytics call, it assumes window is available 😄 . We can instead do:
```
import { recordTracksEvent } from 'state/analytics/actions';
//...
context.store.dispatch( recordTracksEvent( 'calypso_wpcom_masterbar_plugins_add_click' ) );
```